docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+services:
+  gokeyless:
+    platform: linux/amd64
+    container_name: gokeyless
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./tests/testdata/:/gokeyless/config
+    environment:
+      # - KEYLESS_HOSTNAME=keyserver.keyless.com
+      # - KEYLESS_ZONE_ID=5c7004f5221ba2a24c998bd609244a39
+      # - KEYLESS_ORIGIN_CA_API_KEY=ORIGIN_CA_API_KEY_HERE
+      - KEYLESS_AUTH_CERT=/gokeyless/config/server.pem
```

Comment:
I might be missing something, is the code capable of reading this env vars?

Reply:
Yeap, it's hooked in around this https://github.com/cloudflare/gokeyless/blob/master/cmd/gokeyless/gokeyless.go#L157

Reply:
ah, the prefix killing searchability!